OAuthExampleApplication/MainForm.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Configuration;
+using System.Linq;
+using System.Windows.Forms;
+using System.Xml.Linq;
+
+namespace OAuth2ExampleApp

```

Comment:
Turn into namespace statement. This code's structure should be aligned with the structure in the Example Apps code.